cars.py

```diff
@@ -1,10 +1,50 @@
+"""This program describes the existing cars features
+and designs prototype of the new car based on these features"""
+
+
 class Car1:
-    pass
+    """This class represents the car with some tyres and engine"""
+    def __init__(self):
+        """Define the characteristics of Car1 tyres and engine"""
+        self.wheel_type = 'Continental'
+        self.engine_type = '1HZ'
+
+    def get_wheel_type(self):
+        """Provide Car1 tyres type"""
+        return self.wheel_type
+
+    def get_engine_type(self):
+        """Provide Car1 engine type"""
+        return self.engine_type
 
 
 class Car2:
-    pass
+    """This class represents the car with some tyres and engine too"""
+    def __init__(self):
+        """Define the characteristics of Car2 tyres and engine"""
+        self.wheel_type = 'Pirelli'
+        self.engine_type = '1HD-T'
+
+    def get_wheel_type(self):
+        """Provide Car2 tyres type"""
+        return self.wheel_type
+
+    def get_engine_type(self):
+        """Provide Car2 engine type"""
+        return self.engine_type
 
 
 class NewCar:
-    pass
\ No newline at end of file
+    """This class represents the prototype of the car
+    with merged Car1 and Car2 features"""
+    @staticmethod
+    def get_wheel_type():
+        """Provide the Car1 tyres type"""
+        car = Car1()
```

Comment:
Do you create a new car just to take wheel from it?

Reply:
In recent realization I've used static methods only.